docs/openapi.json

```diff
@@ -199,27 +199,23 @@
             "required": true,
             "schema": {
               "properties": {
-                "name": {
-                  "description": "Workflow name. If empty name will be generated.",
-                  "type": "string"
-                },
                 "parameters": {
                   "description": "Workflow parameters.",
                   "type": "object"
                 },
-                "specification": {
-                  "description": "Yadage specification in JSON format.",
+                "reana_yaml": {
+                  "description": "Workflow specification in JSON format.",
                   "type": "object"
                 },
-                "type": {
-                  "description": "Workflow type.",
+                "workflow_name": {
+                  "description": "Workflow name. If empty name will be generated.",
                   "type": "string"
                 }
               },
               "required": [
-                "specification",
-                "type",
-                "name"
+                "reana_yaml",
+                "workflow_name",
+                "parameters"
```

Comment:
Shall we take this occasion and standardise various concepts as:

- reana specification
- workflow specification
- input parameters
- operational parameters
- engine parameters
- ...

here in the schemas and everywhere?

See also https://github.com/reanahub/reana-workflow-controller/pull/124#pullrequestreview-150989990